<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QMainWindow" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(&quot;D:/Kolya/Term 8/guiv/GUI/icons/23.jpg&quot;);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_kolo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1271</width>
      <height>781</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Small Semibold</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_Par30_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-50</y>
       <width>441</width>
       <height>831</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_v1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>241</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-repeat: no-repeat;

background-image: url(:/icons/icons/WhatsApp Image 2023-06-02 at 21.30.35.jpeg);
background-size: cover;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_v3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>241</width>
        <height>164</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/icons/icons/WhatsApp Image 2023-06-02 at 21.30.13.jpeg);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_v2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>241</width>
        <height>157</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/icons/icons/land.jpeg);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_v4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>650</y>
        <width>241</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/icons/icons/WhatsApp Image 2023-06-02 at 21.27.29 (1).jpeg);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_v4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>700</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Small Semibold</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QpushButton#pushButton_v4{
background-color:ffffff;
border-radius: 20px;
}

QPushButton#pushButton_v4:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682,x2:1,y2:0.477, stop:0 rgba(19, 17, 87, 1), stop: 1 rgba(105, 118, 132,226));

border-radius: 20px
}

QPushButton#pushButton_v4:pressed{
padding-left:10px;
paddind-top:10px;
background-color:rgba(19, 17, 87, 1);
border-radius: 10px;
}
</string>
      </property>
      <property name="text">
       <string>video 4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_v3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>510</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Small Semibold</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QpushButton#pushButton_v3{
background-color:ffffff;
border-radius: 20px;
}

QPushButton#pushButton_v3:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682,x2:1,y2:0.477, stop:0 rgba(19, 17, 87, 1), stop: 1 rgba(105, 118, 132,226));

border-radius: 20px
}

QPushButton#pushButton_v3:pressed{
padding-left:10px;
paddind-top:10px;
background-color:rgba(19, 17, 87, 1);
border-radius: 10px;
}
</string>
      </property>
      <property name="text">
       <string>video 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_v2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>320</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Small Semibold</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QpushButton#pushButton_v2{
background-color:ffffff;
border-radius: 20px;
}

QPushButton#pushButton_v2:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682,x2:1,y2:0.477, stop:0 rgba(19, 17, 87, 1), stop: 1 rgba(105, 118, 132,226));

border-radius: 20px
}

QPushButton#pushButton_v2:pressed{
padding-left:10px;
paddind-top:10px;
background-color:rgba(19, 17, 87, 1);
border-radius: 10px;
}


</string>
      </property>
      <property name="text">
       <string>video 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_v1">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Small Semibold</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QpushButton#pushButton_v1{
background-color:ffffff;
border-radius: 20px;
}

QPushButton#pushButton_v1:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682,x2:1,y2:0.477, stop:0 rgba(19, 17, 87, 1), stop: 1 rgba(105, 118, 132,226));

border-radius: 20px;
}

QPushButton#pushButton_v1:pressed{
padding-left:10px;
paddind-top:10px;
background-color:rgba(19, 17, 87, 1);
border-radius: 20px;
}


</string>
      </property>
      <property name="text">
       <string>video 1</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>439</x>
       <y>59</y>
       <width>831</width>
       <height>631</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">gridline-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(28, 0, 47, 255), stop:1 rgba(255, 255, 255, 255));
selection-background-color: rgb(70, 51, 118);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_repv">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>771</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(22, 4, 50);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>771</width>
         <height>581</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>651</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255,255);</string>
     </property>
     <property name="text">
      <string>Video server and client project</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>730</y>
       <width>791</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Small Semibold</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255,255);</string>
     </property>
     <property name="text">
      <string>By: Shady Mohamed, Ahmed Khairi, Raneem Abdelwahab, Aya Mohamed, Kareem Akram, Youssef Shalaby</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="designer.qrc"/>
 </resources>
 <connections/>
</ui>
